bonfire/bonfire.py

```diff
@@ -1016,6 +1038,170 @@ def _cmd_apps_what_depends_on(
     print("\n".join(found) or f"no apps depending on {component} found")
 
 
+@reservation.command("create")
+@click.option(
+    '--requester',
+    '-r',
+    prompt='Requester',
+    type=str,
+    help='Name of the user requesting a reservation',
+)
+@click.option(
+    '--duration',
+    '-d',
+    prompt='Duration',
+    type=str,
+    default='1h',
+    help='Duration of the reservation',
+)
+@options(_timeout_option)
+def _create_new_reservation(requester, duration, timeout):
+    def _err_handler(err):
+        msg = f"reservation failed: {str(err)}"
+        _error(msg)
+
```

Comment:
Maybe for average/"non-bot" users: check if requester already has a reservation -- if so warn them and ask "Do you really want to reserve another one?"

We should have a cli flag so that PR checks and bots don't have to answer this check

Reply:
Added a flag (--bot/-b) that will ignore the existing reservation check